deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkAnnotatedElement.java

```diff
@@ -52,8 +52,14 @@ String getUri() {
     return annotationType;
   }
 
-  TypeElement getActivityElement() {
-    return activityElement;
+  TypeElement getAnnotatedElement() {
+    return annotatedElement;
+  }
+
+  /** Returns the package name for the annotated TypeElement. */
+  String getPackageName() {
+    String qualifiedName = annotatedElement.getQualifiedName().toString();
```

Comment:
also - you can do this in a separate class/method since this is not specific to this class